docs/docs/concepts/functional_api.md

```diff
@@ -123,7 +123,7 @@ def workflow(topic: str) -> dict:
 
     ```pycon
     {'write_essay': 'An essay about topic: cat'}
-    {'__interrupt__': (Interrupt(value={'essay': 'An essay about topic: cat', 'action': 'Please approve/reject the essay'}, resumable=True, ns=['workflow:f7b8508b-21c0-8b4c-5958-4e8de74d2684'], when='during'),)}
```

Comment:
If you're changing it to not be a list you should remove the tuple surrounding this one, right?